docs/elasticsearch-spec.asciidoc

```diff
@@ -50,6 +50,45 @@ spec:
 
 For more information on Elasticsearch settings, see https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html[Configuring Elasticsearch].
 
+[id="{p}-volume-claim-templates"]
+=== Volume Claim Templates
+
+By default the operator creates a `PVC` with a capacity of 1Gi for every Pod in an Elasticsearch cluster. This is to ensure that there is no data loss if a Pod is deleted.
+
+You can customize the volume claim templates used by Elasticsearch to adjust the storage to your needs:
+
+[source,yaml]
+----
+spec:
+  nodes:
+  - volumeClaimTemplates:
+    - metadata:
+        name: elasticsearch-data
```

Comment:
This name always needs to be the same, correct? It may be worth calling that out if so